src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -137,22 +273,41 @@ protected internal override void Write(int pinNumber, PinValue value)
 
         protected override void Dispose(bool disposing)
         {
-            foreach (int pin in _pinNumberToSafeLineHandle.Keys)
+            _disposing = disposing;
+
+            if (_pinNumberToEventHandler != null)
             {
-                if (_pinNumberToSafeLineHandle.TryGetValue(pin, out SafeLineHandle pinHandle))
+                foreach (var kv in _pinNumberToEventHandler)
```

Comment:
nit: don't use var here.

Reply:
For my knowledge it would be great if some information/comparison given about advantage of using real type instead of var

Reply:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

> 10. We only use `var` when it's obvious what the variable type is (e.g. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).

Essentially, use `var` when the code reviewer/reader can easily tell what the type is (statically). This  is useful since it removes redundant type information. Otherwise, explicitly use the type name. The advantage is that it makes reviewing code easier (by making it easy to understand and follow).

Reply:
@ahsonkhan the kv is irrelevant since she assigns the Key and Values to locals with explicit type line later. I'm not seeing much value in typing that